godot-core/src/builtin/vectors/vector3.rs

```diff
@@ -411,6 +438,29 @@ mod test {
         assert_eq_approx!(a.coord_max(b), Vector3::new(1.2, 5.6, 5.6));
     }
 
+    #[test]
+    fn test_slerp() {
+        // The halfway point of a slerp operation on two vectors on a circle is the halfway point of
+        // the arc length between the two vectors.
+        let vector_from = Vector3::new(0.0, 2.0, 0.0);
+        let vector_to = Vector3::new(2.0, 0.0, 0.0);
+        let vector_in_between = Vector3::new((2.0 as real).sqrt(), (2.0 as real).sqrt(), 0.0);
```

Comment:
Instead of `(2.0 as real)`, use `real!(2.0)`.

Could also extract the expression as `sqrt_2` or so.